observer.py

```diff
@@ -201,7 +210,18 @@ async def run_coingecko_get_price():
                 [x for x in symbol_to_id_mapping]
             )
 
-    await asyncio.gather(run_alerts(), run_coingecko_get_price())
+    async def run_crosschain_get_price():
+        # TODO: add support for other networks when live
+        if args.network == "devnet":
+            # sleep for 2 seconds to allow pyth-observer to load products for the first time
+            await asyncio.sleep(2)
```

Comment:
not sure i understand why this sleep is necessary. I think `get_crosschain_prices` works regardless of whether or not the products are loaded from solana right?

Reply:
good catch, previously `get_crosschain_prices` took in the products as params but we switched to calling the api and hence this is unnecessary now